src/sidebar/util/test/oauth-client-test.js

```diff
@@ -192,7 +191,7 @@ describe('sidebar/util/oauth-client', () => {
         fakeWindow.open,
         'about:blank',
         'Log in to Hypothesis',
-        'height=430,left=274.5,top=169,width=475'
+        'left=274.5,top=169,width=475,height=430'
```

Comment:
Is this change to exercise a path in which the params are not already alpha-sorted?

Reply:
No, this is just because the params are left in the order that the code generates them, rather than being re-sorted by `queryString.stringify` as they were before.